packages/core/src/Stores/client-store.js

```diff
@@ -2570,8 +2570,10 @@ export default class ClientStore extends BaseStore {
         return !!this.accounts[this.loginid]?.residence;
     }
 
-    get is_pending_proof_of_ownership() {
-        return this.account_status?.authentication?.needs_verification?.includes('ownership');
+    get is_proof_of_ownership_enabled() {
+        if (!this.account_status?.authentication) return false;
+        const { ownership, needs_verification } = this.account_status.authentication;
+        return needs_verification?.includes('ownership') || ownership?.status === 'verified';
```

Comment:
so what happens when proof of ownership status is pending?
Based on the code, it seems route will be disabled?

Reply:
If status is pending, the needs_verifiction array will have `ownership` flag. Hence, the route will not be disabled. The needs_verification array will not have `ownership` only if POO is not triggered for a client or if the POO is verified. Hence the OR condition in the check for enabling POO